lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu
+
+attr_reader :dishes
```

Comment:
Don't expose the dishes hash publicly.  Write methods on `Menu` to allow other objects to query the dishes.